index.js

```diff
@@ -0,0 +1,88 @@
+/* Set Date to Current Date */
+Date.prototype.toDateInputValue = (function() {
```

Comment:
In general I don't think it's good to modify the prototype of JavaScript objects. Was there a reason for doing this?

Reply:
Resolved in Week 2 Branch